### PR TITLE
June 14, 2021 Demo Day Blog Post

### PR DESCRIPTION
**Pull request by andersonfrailey**
This PR adds my blog post from the June 14, 2021 demo day. I'll need to update the link in the post when the recording is posted, but I wanted to get the main text up for review.

### CODE REVIEW

**Comment by andersonfrailey** [comment]
Latest commit adds the link for the video

**Comment by jdebacker** [comment]
@andersonfrailey Thanks for including the video link, can you post a screenshot of the page?  You can create it following the steps in the [contribution guide](https://github.com/PSLmodels/blog#how-to-contribute).

File: _posts/2021-06-14-demo-day-13-taxbrain-python-api.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+in just three lines of code:
+
+```python
+from taxbrain import TaxBrain
+
+tb = TaxBrain(START_YEAR, END_YEAR, use_cps=True, reform=REFORM_POLICY)
+tb.run()
+```
+
+Where `START_YEAR` and `END_YEAR` and the first and last years, respectively,
```
Comment (jdebacker) [inline _posts/2021-06-14-demo-day-13-taxbrain-python-api.md:34]: "are the first..."

File: _posts/2021-06-14-demo-day-13-taxbrain-python-api.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+of the analysis; `use_cps` is a simple boolean indicator that you want to use
+The CPS-based microdata file prepared for use with Tax-Calculator; and `REFORM_POLICY`
```
Comment (jdebacker) [inline _posts/2021-06-14-demo-day-13-taxbrain-python-api.md:36]: "The" doesn't need to be capitalized here.

File: _posts/2021-06-14-demo-day-13-taxbrain-python-api.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+results and an included report template to write a report summarizing the findings
+of your simulation. The reports include tables and figures similar to what you
+may find in similar write ups by the Joint Committee on Taxation or Tax Policy Center,
+and all you need is one line of code:
+
+```python
+report(tb, name='Biden Proposal', outdir='biden', author='Anderson Frailey')
+```
+
+The above code will save a PDF copy of the report in a directory called `biden`
```
Comment (jdebacker) [inline _posts/2021-06-14-demo-day-13-taxbrain-python-api.md:73]: This is a cool feature.  Do you want to include a screenshot of the report?  Also, might be worth mentioning how the report summarizes the major changes automatically -- that's super cool.

**Comment by jdebacker** [comment]
@andersonfrailey I had a few small suggestions on you post.  But it would be helpful if you also changed the formatting a bit to be consistent with other posts.  In particular, each sentence should be it's own line (see another recent post [here](https://github.com/PSLmodels/blog/pull/50/files)).

**Comment by andersonfrailey** [comment]
Thanks for the review, @jdebacker! It's a bit long for a screenshot, but if you click on the image you can zoom in and see everything. Latest commit includes the grammar fixes and a screen shot of the report.
Screen Shot 2021-07-16 at 18 28 41

**Comment by jdebacker** [comment]
LGTM -- thank you @andersonfrailey!

**Pull request merged**